converter/internal/otelcolconvert/testdata/transform.yaml

```diff
@@ -0,0 +1,33 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+
+processors:
+  transform:
```

Comment:
Can we use a real-world example here? This is unlikely to be a common usage of the transform processor, I feel

Reply:
Yeah, I've added the example from the [upstream component's README](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/transformprocessor/README.md).